docs/src/modules/concepts/pages/concepts.adoc

```diff
@@ -172,46 +172,54 @@ Akka provides a basket of interoperable components that can be used to design an
 
 == Component interoperability
 
-Akka components are able to interoperate with one another regardless of where the component may be executing. While you create systems that define the relationships between each of the components in code, at runtime instances of each component can run on different nodes. The Akka runtime provides location transparency, automating the routing of a call from one component to another even if they reside in different locations on different networks.
+Systems often rely on distributed components that need to work together. In Akka, components such as Agents, Workflows and Entities interact in ways that support flexibility, scale, and resilience. The aim is to help you build systems that are easy to reason about and maintain, even when deployed across different environments.
 
-There are two ways to achieve interoperability between Akka components and the outside world.
+In Akka, component relationships are defined in code. At runtime, the platform handles the details. Messages are routed automatically, without requiring you to manage network paths or address resolution. This is known as _location transparency_. It means components can communicate without knowing where the other components are running.
 
+Akka supports two primary ways for components to interact, either with each other or with the outside world.
 
 [cols="1,3", options="header"]
 [.compact]
 |===
 |Client Type |Description
 
 |xref:java:component-and-service-calls.adoc[ComponentClient]
-|One component can directly invoke another component. Akka treats these direct invocations as non-blocking, asynchronous messages, meaning that the invoking component immediately returns after making the invocation and can invoke any component regardless of its location.
+|One component can invoke another using a direct call. These invocations are non-blocking and asynchronous. The invoking component continues its work without waiting for a reply. This helps maintain responsiveness in the system.
 
-For example, a Workflow component handling a long-running business transaction can invoke an Agent component to perform a task, and the Workflow can continue processing without waiting for the Agent to complete its work.
+A common example is a Workflow that performs a long-running process and invokes an Agent to handle a specific task. The Workflow can move on while the Agent does its work in parallel.
```

Comment:
Not really, a workflow can only run one step at a time. You can achieve it by spawning several calls inside the same step, and compose them with CompletionStage operations, but this goes into details that shouldn't be in concepts section.